src/patternfly/components/Wizard/wizard.scss

```diff
@@ -0,0 +1,398 @@
+@import "../../patternfly-imports";
+
+.pf-c-wizard {
+  --pf-c-wizard--GridTemplateRows: auto auto 1fr auto;
+  --pf-c-wizard--md--GridTemplateRows: auto 1fr auto;
+  --pf-c-wizard--md--GridTemplateColumns: 300px 1fr;
+  --pf-c-wizard--Width: calc(100% - var(--pf-global--spacer--xl) * 2);
+  --pf-c-wizard--Height: calc(100% - var(--pf-global--spacer--xl) * 2);
+  --pf-c-wizard--BoxShadow: var(--pf-global--BoxShadow--lg);
+  --pf-c-wizard--children--BackgroundColor: var(--pf-global--BackgroundColor--100);
+
+  @media screen and (min-width: $pf-global--breakpoint--md) {
+    --pf-c-wizard--GridTemplateRows: var(--pf-c-wizard--md--GridTemplateRows);
+  }
+
+  // Header
+  --pf-c-wizard__header--PaddingTop: var(--pf-global--spacer--lg);
+  --pf-c-wizard__header--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-c-wizard__header--PaddingBottom: var(--pf-global--spacer--lg);
+  --pf-c-wizard__header--PaddingLeft: var(--pf-global--spacer--xl);
+
+  // Close
+  --pf-c-wizard__close--Top: calc(var(--pf-global--spacer--lg) - var(--pf-global--spacer--form-element));
+  --pf-c-wizard__close--Right: calc(var(--pf-global--spacer--xl) - var(--pf-global--spacer--md));
+  --pf-c-wizard__close--FontSize: var(--pf-global--FontSize--xl);
+
+  // Title
+  --pf-c-wizard__title--PaddingRight: var(--pf-global--spacer--xl);
+
+  // Description
+  --pf-c-wizard__description--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-wizard__description--Color: var(--pf-global--Color--400);
+
+  // Steps
+  --pf-c-wizard__steps--md--BoxShadow: var(--pf-global--BoxShadow--md-right);
+
+  // Toggle
+  --pf-c-wizard__toggle--PaddingTop: var(--pf-global--spacer--lg);
+  --pf-c-wizard__toggle--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-c-wizard__toggle--PaddingBottom: var(--pf-global--spacer--lg);
+  --pf-c-wizard__toggle--PaddingLeft: var(--pf-global--spacer--xl);
+  --pf-c-wizard__toggle--BoxShadow: var(--pf-global--BoxShadow--md-bottom);
+  --pf-c-wizard__toggle--ZIndex: var(--pf-global--ZIndex--sm);
+
+  // Toggle Label
+  --pf-c-wizard__toggle-label--MarginRight: var(--pf-global--spacer--sm);
```

Comment:
should this instead have a (max-width of 100% - width of icon), so that the text truncates rather than cutting off? Not sure what design calls for.